src/event.rs

```diff
@@ -201,6 +201,12 @@ pub enum WindowEvent {
     ///
     /// For more information about DPI in general, see the [`dpi`](crate::dpi) module.
     HiDpiFactorChanged(f64),
+
+    /// The system window theme has changed.
+    ///
+    /// Applications might wish to react to this to change the theme of the content of the window
+    /// when the system changes the window theme.
+    ThemeChanged(Theme),
```

Comment:
We should probably mention that it is currently only implemented for windows.

Reply:
Thanks, have done so!